### PR TITLE
country code URL and add comma in population

### PR DESCRIPTION
**Pull request by Vishal-progammer**

### CODE REVIEW

File: config/dev.exs [hunk 1]
```diff
@@ -19,7 +19,7 @@ config :country_finder, CountryFinder.Repo,
 config :country_finder, CountryFinderWeb.Endpoint,
   # Binding to loopback ipv4 address prevents access from other machines.
   # Change to `ip: {0, 0, 0, 0}` to allow access from other machines.
-  http: [ip: {127, 0, 0, 1}, port: 4000],
+  http: [ip: {127, 0, 0, 1}, port: 4003],
```
Comment (nittin-shankar) [inline config/dev.exs]: Change this back to 4000

File: lib/country_finder/country.ex [hunk 2]
```diff
@@ -12,6 +12,7 @@ defmodule CountryFinder.Country do
     :flag_png,
     :region,
     :capital,
-    :population
+    :population,
+    :countrycode
```
Comment (nittin-shankar) [inline lib/country_finder/country.ex]: Rename this as `country_code`. It's more readable in this way.

File: lib/country_finder/country_client.ex [hunk 3]
```diff
@@ -2,7 +2,7 @@ defmodule CountryFinder.CountryClient do
   @behaviour CountryFinder.CountryClientBehaviour
 
   def all_countries() do
-    url = "https://restcountries.com/v3.1/all?fields=flags,name,population,region,capital"
+    url = "https://restcountries.com/v3.1/all?fields=flags,name,population,region,capital,cca2"
```
Comment (nittin-shankar) [inline lib/country_finder/country_client.ex]: Use `cca3` instead of `cca2`.

**Review by nittin-shankar — CHANGES_REQUESTED** [review]
Good job @Vishal-progammer. Very happy that you completed it. I just have a few changes for you to make.

File: config/dev.exs (hunk 1 above)
Comment (Vishal-progammer) [reply config/dev.exs]: done

File: lib/country_finder/country_client.ex (hunk 3 above)
Comment (Vishal-progammer) [reply lib/country_finder/country_client.ex]: done

**Pull request closed without merging**